tests/test_sklearn_tfidf_vectorizer_converter_pipeline.py

```diff
@@ -20,8 +20,8 @@ class TestSklearnTfidfVectorizerPipeline(unittest.TestCase):
     def common_test_model_tfidf_vectorizer_pipeline_cls(
             self, kind=None, verbose=False):
         if kind == 'stop':
-            if ort_version.startswith('1.4'):
-                # regression with stopwords in onnxruntime 1.4
+            if ort_version.startswith('1.4') or ort_version.startswith('1.5'):
```

Comment:
is it true that ort_version >= 1.4 so there is no issue for 1.6 which may be two months later.